### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by DStatWriter**
Signed-off-by: Daniel Statham <Daniel.Statham@Broadcom.com>

### CODE REVIEW

**Comment by zeibura** [comment]
Can profiles be renamed, because I think @zdmullen told us they couldn't in the last PR.

**Comment by DStatWriter** [comment]
> Can profiles be renamed, because I think @zdmullen told us they couldn't in the last PR.

Ive asked for clarification

File: Readme.md [hunk 1]
```diff
@@ -64,9 +64,12 @@ If you do not add credentials to your profile, a credential prompt displays when
...
     - **False**: Accept connections with self-signed certificates.
     
-Your profile is now available in the panel on the left. You can also use this profile in Zowe CLI directly.
-
-If you have multiple profiles, you might wish to delete some once you no longer need them. To do so, click on the trash can icon on the same line as the profile name.
+    Your profile is now available in the panel on the left. You can also use this profile in Zowe CLI directly.
+    
+**Rename or Delete Profiles**   
+    
+- To change the profile name, click the pencil icon next to the profile you wish to change. Ensure that the new profile name is unique.
```
Comment (Alexandru-Dumitru) [inline Readme.md:71]: @DStatWriter 
I can confirm that profiles cannot be renamed from the explorer. Also there is no pencil icon present to do so. You would have to correct these lines. Thanks

Comment (zeibura) [reply Readme.md:71]: zdmullen 2 days ago Member
I would just say that you can edit just the names of the data sources (configurations). We have not yet added the ability to edit profiles here.

How does the user edit the name of a configuration? Pencil icon next to the configuration name in the tree?

File: Readme.md [hunk 2]
```diff
@@ -64,9 +64,12 @@ If you do not add credentials to your profile, a credential prompt displays when
     - **True**: Reject connections with self-signed certificates.
     - **False**: Accept connections with self-signed certificates.
     
-Your profile is now available in the panel on the left. You can also use this profile in Zowe CLI directly.
-
-If you have multiple profiles, you might wish to delete some once you no longer need them. To do so, click on the trash can icon on the same line as the profile name.
```
Comment (Alexandru-Dumitru) [inline Readme.md:69]: Also, the tool tip on the trash can states `Remove Profile`, as not to confuse the users that the profile is deleted from the system as well. Please see:
https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/c9ae4f159730435e70c7c6a4da74d90cd27b2ca0/package.json#L60-L61

File: Readme.md (hunk 1 above)
Comment (Alexandru-Dumitru) [reply Readme.md:71]: Yes, that is correct. That icon remained in place and has the same purpose.

File: Readme.md (hunk 2 above)
Comment (zeibura) [reply Readme.md:69]: I've changed the wording to better reflect this

File: Readme.md (hunk 1 above)
Comment (zeibura) [reply Readme.md]: OK, reworded and moved under the "working with configurations" section.

Comment (DStatWriter) [reply Readme.md]: Great thanks Zeb

**Pull request merged**